docs/admin-manual/configuration-macros.rst

```diff
@@ -4353,6 +4353,13 @@ See (:ref:`admin-manual/ep-policy-configuration:power management`). for more det
     defaults to 1200 (seconds), controls how often the start
     polls the docker daemon for this information.
 
+:macro-def:`DOCKER_LOG_DRIVER_NONE[TARTD]`
```

Comment:
Shouldn't this be "[STARTER]", not "[TARTD]" ?   What does this do anyway?  The commit associated with adding all these square bracket sections just says "Manual Improvements".

Reply:
It should be [STARTD].  It makes an index entry under the "STARTD" subheading (i.e. https://htcondor--2031.org.readthedocs.build/en/2031/genindex.html#S ) 

See the manual style guide: https://github.com/htcondor/htcondor/blob/main/docs/STYLE.md for all the custom markup.

Reply:
Please explain why it should be [STARTD] and not [STARTER].  This new config knob is referenced in `DockerAPI:CreateContainer()` which is only invoked by the condor_starter.  This makes sense, as it is the starter that launches containers.  Once we add support for containers in local universe, this new config knob will still be referenced even though no condor_startd involved, right?

Reply:
The reason this docker knob is in STARTD is so that it is textually next to the rest of the DOCKER knobs, all of which happen to be in the STARTD section.